2017-bae-europe/eu_future/eu_future.xml

```diff
@@ -4834,87 +4690,12 @@
     <name>France</name>
     <config><NullRegion/></config>
     <institution>
-        <name>facilities</name>
-        <config>
-            <DeployInst>
-                <prototypes>
-                    <val>uox_pool</val>
-                    <val>mox_pool</val>
-                    <val>enrichment</val>
-                    <val>nat_u_source</val>
-                    
-                    <!-- Realistic -->
-                    <val>uox_reprocessing</val>
-                    <val>uox_mixer</val>
-
-                    <!-- FR transition infinite -->
-                    <val>inf_uox_reprocessing</val>
-                    <val>inf_uox_mixer</val>
-                    <val>inf_mox_reprocessing</val>
-                    <val>inf_mox_mixer</val>
-                </prototypes>
-
-                <build_times>
-                    <val>1</val>
-                    <val>1</val>
-                    <val>1</val>
-                    <val>1</val>
-                    
-                    <!-- Realistic from 1976 (carre_overview_2009)-->
-                    <val>72</val>
-                    <val>72</val>
-
-                    <!-- FR transition infinite from 2040 -->
-                    <val>840</val>
-                    <val>840</val>
-                    <val>840</val>
-                    <val>840</val>
-                </build_times>
-
-                <n_build>
-                    <val>1</val>
-                    <val>1</val>
-                    <val>1</val>
-                    <val>1</val>
-
-                    <!-- Realistic from 1976 (carre_overview_2009)-->
-                    <val>1</val>
-                    <val>1</val>
-
-                    <!-- FR transition infinite from 2040 -->
-                    <val>1</val>
-                    <val>1</val>
-                    <val>1</val>
-                    <val>1</val>
-                </n_build>
-
-                <lifetimes>
-                    <val>9999</val>
-                    <val>9999</val>
-                    <val>9999</val>
-                    <val>9999</val>
-                    
-                    <!-- Realistic (840-72 = 728)-->
-                    <val>728</val>
-                    <val>728</val>
-
-                    <!-- FR transition infinite from 2040 -->
-                    <val>9999</val>
-                    <val>9999</val>
-                    <val>9999</val>
-                    <val>9999</val>
-                </lifetimes>
-            </DeployInst>
-        </config>
-    </institution>
-
-    <institution>
       <name>France_government</name>
       <config>
         <DeployInst>
 
           <prototypes>
-<val>CHINON A-2</val>
+  <val>CHINON A-2</val>
```

Comment:
Sorry I've been sloppy with the indents.. Luckily I found an auto indent function and will use it prior to submission